script.artwork.downloader/resources/settings.xml

```diff
@@ -9,7 +9,7 @@
         <setting type="sep" />
         <!--
         <setting label="32023" type="bool"      id="service_startup"    default="false"/>
-        <setting label="32019" type="labelenum" id="service_startupdelay" subsetting="true" values="00|10|20|30" default="--" visible="eq(-1,true)"/>
+        <setting label="32019" type="labelenum" id="service_startupdelay" subsetting="true" values="00|10|20|30" default="40" visible="eq(-1,true)"/>
```

Comment:
Default is out of range from the setting, please correct... ie 00|10|20|30|**40**

Otherwise LGTM

Reply:
Done.